integration-test/1983-china-south-sea.py

```diff
@@ -0,0 +1,32 @@
+# -*- encoding: utf-8 -*-
+from . import FixtureTest
+
+
+class NaturalEarth(FixtureTest):
+    def test_claim_boundary(self):
+        import dsl
+
+        z, x, y = 1, 0, 0
+
+        self.generate_fixtures(
+            dsl.way(1, dsl.tile_diagonal_dexter(z, x, y), {
+                'featurecla': 'Claim boundary',
+                'name': 'Chinese claim',
+                'adm0_usa': -1,
+                'scalerank': 7,
+                'fclass_cn': 'International boundary (verify)',
+                'fclass_us': 'Unrecognized',
+                'min_zoom': 7.0,  # min_zoom is set to 5 in the source data
+                'source': 'naturalearthdata.com',
+            }),
+        )
+        for zoom in range(1, 17):
+            self.assert_has_feature(
+                zoom, x, y, 'boundaries', {
+                    'kind': 'disputed_claim',
```

Comment:
@nvkelso according to the output, the kind is `disputed_claim` (not `unrecognized_country`)

Reply:
That's on me in Natural Earth. I will correct the data upstream and provide you a new ZIP to reference in the assets file.